nautobot_circuit_maintenance/urls.py

```diff
@@ -3,32 +3,19 @@
 from django.templatetags.static import static
 from django.urls import path
 from django.views.generic import RedirectView
+from nautobot.core.views.routers import NautobotUIViewSetRouter
 from nautobot.extras.views import ObjectChangeLogView
 
 from . import views
-from .models import CircuitImpact, CircuitMaintenance, Note, NotificationSource
+from .models import CircuitImpact, NotificationSource
+
+router = NautobotUIViewSetRouter()
+router.register("circuit-maintenances", views.CircuitMaintenanceUIViewSet)
 
 urlpatterns = [
     # Overview
-    path("circuit-maintenances/overview/", views.CircuitMaintenanceOverview.as_view(), name="circuitmaintenance_overview"),
-    #  Maintenance
-    path("circuit-maintenances/", views.CircuitMaintenanceListView.as_view(), name="circuitmaintenance_list"),
-    path("circuit-maintenances/add/", views.CircuitMaintenanceEditView.as_view(), name="circuitmaintenance_add"),
-    path("circuit-maintenances/import/", views.CircuitMaintenanceBulkImportView.as_view(), name="circuitmaintenance_import"),
-    path("circuit-maintenances/edit/", views.CircuitMaintenanceBulkEditView.as_view(), name="circuitmaintenance_bulk_edit"),
-    path(
-        "circuit-maintenances/delete/", views.CircuitMaintenanceBulkDeleteView.as_view(), name="circuitmaintenance_bulk_delete"
-    ),
-    path("circuit-maintenances/<uuid:pk>/", views.CircuitMaintenanceView.as_view(), name="circuitmaintenance"),
-    path("circuit-maintenances/<uuid:pk>/edit/", views.CircuitMaintenanceEditView.as_view(), name="circuitmaintenance_edit"),
-    path(
-        "circuit-maintenances/<uuid:pk>/delete/", views.CircuitMaintenanceDeleteView.as_view(), name="circuitmaintenance_delete"
-    ),
     path(
-        "circuit-maintenances/<uuid:pk>/changelog/",
-        ObjectChangeLogView.as_view(),
-        name="circuitmaintenance_changelog",
-        kwargs={"model": CircuitMaintenance},
+        "circuit-maintenances/overview/", views.CircuitMaintenanceOverview.as_view(), name="circuitmaintenance_overview"
```

Comment:
Possibly migrate this view into a custom action on `CircuitMaintenanceUIViewSet` as well? Not blocking for this PR though.